### DISCUSSION TITLE
pico run apk crash

### DISCUSSION THREAD

**Discussion by thisleo** [korejan/ALVR#30] [Q&A] [answered]
Running ALXR on pico prompts crashes

**Comment by korejan** [comment, marked as answer]
Pico Neo 3 Link variant works, I need someone who owns a non-link variant to work with me or some how get donated or loaned one.

**Reply by AxelTheAvali** [reply]
i own the pico neo 3 pro eye however i am not a developer so i can only run whatever you create and send back the logs

**Reply by thisleo** [reply]
the new version works well on pico neo3
